src/Security/PasswordStrengthService.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Security;
```

Comment:
Eventueel uitleggen waarom je https://github.com/symfony/symfony/blob/7.0/src/Symfony/Component/Validator/Constraints/PasswordStrengthValidator.php niet rechtstreeks gebruikt. Omdat het private static method is?

Reply:
Dat lijkt gewoon een kopie van Symfony code? Niet beter om dat een validator te gebruiken in uw [Ajax call](https://github.com/sumocoders/Framework-User-Implementation-Example/commit/b0adf9971b3545425e22c987fd6f274990b39654#diff-376b6250bda1cf513fff9b7ec946845eb316ccae46529a186a143888801a3e9bR10)? Zoals eerste voorbeeld op https://symfony.com/doc/current/components/validator.html (maar dan met PasswordStrength Assert). Want als Symfony hun "strength" update, dan gaan we onze code ook moeten updaten.

Reply:
@jonasdekeukelaere het probleem daarmee is dat je niet de waarde 1 -> 4 terug krijgt. Daphne heeft die nodig voor de frontend
=

Reply:
@tijsverkoyen fixed c1224ba632eaaf2c7549647891050770a5c0d762

Reply:
Zou het dan geen idee zijn om een PR naar symfony te doen om die waarde wel terug te geven? Al is het maar die method public maken? Als je uitlegt dat het handig kan zijn om password strength indicator te kunnen bouwen?

Alternatief is eventueel de strength in de "message" verwerken zoals bij https://github.com/symfony/symfony/blob/78c6ceb5cd0560346b6765849f142c1c451fd1a8/src/Symfony/Component/Validator/Constraints/File.php#L57. Maar oprecht geen idee hoe dat werkt.

Want wat Jonas zegt klopt volledig dat we dat nu zelf gaan moeten onderhouden. Plus het kan ook zijn dat er verschillen optreden op termijn.

Ale, for now mergen, maar ook PR doen. En als die gemerged zou worden dan kunnen we "onze" code schrapen.

Reply:
symfony PR aangemaakt https://github.com/symfony/symfony/pull/54405

Reply:
Top, er is wel al feedback. Maybe best eens checken.